src/Microsoft.AspNetCore.Session/SessionOptions.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using Microsoft.AspNetCore.Session;
+using Microsoft.AspNetCore.Http;
```

Comment:
Sort. There's a VS feature to sort for you, but you'll have to enable the option for putting `Systetm.` first.